types/selenium-webdriver/chrome.d.ts

```diff
@@ -22,6 +22,15 @@ export class Driver extends webdriver.WebDriver {
   static createSession(
       opt_config?: Options|webdriver.CreateSessionCapabilities,
       opt_service?: remote.DriverService|http.Executor): Driver;
+
+  /**
+   * Sends a DevTools command to change the browser's download directory.
+   *
+   * @param {string} path The desired download directory.
+   * @return {!Promise<void>} A promise that will be resolved when the command
+   *     has finished.
+  */
+  setDownloadPath(path: String): Promise<void>;
```

Comment:
I think that `path` should be of type `string` rather than `String`. Check out [this StackOverflow post](https://stackoverflow.com/a/14727461) for why.